src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -61,6 +61,22 @@
                  description="${%enableGerritTriggerDescription}">
             <f:checkbox name="gerritTriggerEnabled" checked="${it.gerritTriggerEnabled}" default="true"/>
         </f:entry>
+        <f:entry title="${%Send notifications to Slack}"
```

Comment:
It would be better if this whole part was an `optionalProperty` that only rendered when the slack plugin is installed.

Reply:
Wasn't sure how to do `optionalProperty` or `optionalBlock` depending on plugin existence.
But changed it so the additional settings are in an optionalBlock based on when checkbox for `Send Notifications to Slack` is set.